QTRecorder/QTRecorder.csproj

```diff
@@ -23,6 +23,7 @@
     <WarningLevel>4</WarningLevel>
     <ConsolePause>false</ConsolePause>
     <UseSGen>false</UseSGen>
+    <MonoBundlingExtraArgs>--registrar=dynamic</MonoBundlingExtraArgs>
```

Comment:
We should update the readme to note this explicitly. Customers are not going to see this.